src/main/java/frc/robot/subsystems/drive/DriveSwerveYAGSL.java

```diff
@@ -21,6 +26,36 @@ public DriveSwerveYAGSL() {
     } catch (Exception e) {
       throw new RuntimeException(e);
     }
+
+    AutoBuilder.configureHolonomic(
+        swerveDrive::getPose, // Robot pose supplier
+        swerveDrive
+            ::resetOdometry, // Method to reset odometry (will be called if your auto has a starting
+        // pose)
+        swerveDrive::getRobotVelocity, // ChassisSpeeds supplier. MUST BE ROBOT RELATIVE
+        swerveDrive::drive, // Method that will drive the robot given ROBOT RELATIVE ChassisSpeeds
+        new HolonomicPathFollowerConfig( // HolonomicPathFollowerConfig, this should likely live in
+            // your Constants class
+            new PIDConstants(5.0, 0.0, 0.0), // Translation PID constants
+            new PIDConstants(5.0, 0.0, 0.0), // Rotation PID constants
+            4.5, // Max module speed, in m/s
+            0.4, // Drive base radius in meters. Distance from robot center to furthest module.
```

Comment:
Are these the defaults?  Do you know what these values mean?  We likely will need to change these to match the actual robot capabilities so we should understand how to calculate these values.

Reply:
As discussed, these are optional.